Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md

```diff
@@ -0,0 +1,45 @@
+---
+versionFrom: 8.0.0
+---
+
+# Email Address
+
+`Alias: Umbraco.EmailAddress`
+
+`Returns: String`
+
+Displays an email address.
+
+## Settings
+
+### Enable required checkbox (8.7.0 and below)
+
+If the property is set to mandatory, Umbraco will display a warning label under the property editor when you publish the page this editor is located on and clean the input. This functionality has been deprecated in `8.8.0` and above. Instead, you select the mandatory checkbox when adding the property editor to a Document Type.
+
+## Data Type Definition Example
+
+### 8.8.0 and higher
+![Email Data Type Definition 8.8.0](images/EmailAddress-DataType-v8.8.png)
+
+### Mandatory checkbox example
+![Mandatory Checkbox Example](images/mandatory-checkbox.png)
+
+### 8.0.0 - 8.7.0
+![Email Data Type Definition 8.0.0 - 8.7.0](images/EmailAddress-DataType-v8.png)
+
+## Content Example
+
+![Single email address content example](images/EmailAddress-DataType-Content.png)
+
+## MVC View Example
+
+### Typed
+
+```csharp
+@if (Model.HasValue("email"))
+{
+    var emailAddress = Model.Value<string>("email");
+
+    <p>@email</p>
```

Comment:
Should be the same as the variable created in line 41.

```suggestion
    <p>@emailAddess</p>
```

Reply:
Whoops 😅